src/rantanplan/core.py

```diff
@@ -526,16 +578,23 @@ def get_words(word_list, alternative_syllabification=False):
     :rtype: list
     """
     syllabified_words = []
-    for word in word_list:
+    for index, word in enumerate(word_list):
         if word.is_alpha:
             if '__' in word.tag_:
                 pos, tag = word.tag_.split('__')
             else:
                 pos = word.pos_ or ""
                 tag = word.tag_ or ""
             tags = spacy_tag_to_dict(tag)
-            stressed_word = get_word_stress(word.text, pos, tags,
-                                            alternative_syllabification)
+            # If it's the last word of a verse, mark it so it's always stressed
+            # `is` is used here to be sure it's the same spacy object
+            if word is [w for w in word_list if w.is_alpha][-1]:
+                stressed_word = get_word_stress(word.text, pos, tags,
+                                                alternative_syllabification,
+                                                is_last_word=True)
+            else:
+                stressed_word = get_word_stress(word.text, pos, tags,
+                                                alternative_syllabification)
```

Comment:
Is there a test checking the two branches of that `if`? Does spacy-affixes have any effect on the `is` comparison? I mean, if the last word has clitics, is that still taken into account with `is`?

Reply:
yes, I added a `test_get_scansion_stress_last_word` on test_core :)

Reply:
Oh but I forgot to add one where last_word is False